### DISCUSSION TITLE
东北师范大学（净月校区）

### DISCUSSION THREAD

**Discussion by A1exMinatoooo** [CollegesChat/university-information#82] [University]
仅描述净月校区情况，本部（自由校区）情况略有不同

**宿舍是上床下桌吗**
是。桌子上有安排好的置物架隔断
木门，有阳台，阳台有晾衣杆

**教室和宿舍有没有空调**
宿舍无，大部分教室没有空调

**有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远**
无独立卫浴，澡堂距离决定于住哪个公寓，平均 200-400 m

**有早自习、晚自习吗**
本人经历，大一只有过非常短暂的一段时间，且非强制。各学院自行决定，没有统一规定

**有晨跑吗**
没有，爬

**每学期跑步打卡的要求是多少公里，可以骑车吗**
运动世界校园 APP。仅作额外加分，非强制进行。每学期男生 80 km / 女生 60 km，单次有效里程上限 3 km / 2 km
- 若申请免测 / 缓测，则不需要进行校园跑

**寒暑假放多久，每年小学期有多长**
寒暑假均为 6 周。没有小学期

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
允许，非疫情期间可送至寝室 / 教学楼下。疫情期间需到几个约定俗成的取餐点取餐

**学校附近有地铁站吗**
两校区均有地铁站 / 公交站。主校区：地铁 1 号线【东北师大】，净月校区：轻轨 3 号线【博硕路-东北师大】
- 长春轨道交通 / 公交均支持交通联合
    - 轨道交通可额外使用银联闪付 / 云闪付乘车码 / 支付宝乘车码 / 微信乘车码 / 长春e出行
    - 公交车可使用小程序【同程乘车呗】刷码乘车

**宿舍楼有洗衣机 / 吹风机吗**
宿舍楼每层两侧水房内各有一台公用洗衣机，甩干 0.5 元 / 次，标准洗涤程序 3 元 / 次。洗衣机定期消毒
提供公用吹风机，0.20 元 / 分钟

**校园网怎么样**
供网时间 6:00~24:00，网页 HTTP GET 方式认证，20 元包月有线+无线，同时允许 3 终端在线。速度稳定在 20 Mbps 对等，人品好的时候下载能上到 80 Mbps。教学楼无线速度比宿舍快。有线 IPv6 有点问题，无线 IPv6 可正常接入
- 校园内 Wi-Fi 接入点数量较多，基本上走在楼旁都可接收信号，且在不断新增
- 每间宿舍内均有 AP，提供 4 个 LAN 口（然而是六人寝，打一架吧）

**每天断电断网吗，几点开始断**
断，且断得丧心病狂

- 教学周
    - 周一至周五 `6:00~8:00 | 11:30~13:30 | 15:00~23:00`
    - 周六日 `6:00~23:00`
    - 节假日按周六日进行（如果电工没有忘记调整的话）
- 考试周
    - 周一至周日 `6:00~24:00`
- 假期留校
    - 周一至周日 `6:00~8:00 | 15:00~23:00`

供网时间参考上一条。

**通宵自习有去处吗**
宿舍每层楼均设有自习室，有灯，没插座

**食堂价格贵吗，会吃出异物吗**
食堂价格适中（比本部贵），干净。有清真食堂
- 消费方式
    - 除清真食堂外，仅可使用校园卡，没带可以向周围同学借
    - 请尽量避免使用其他支付方式，食堂管理公司每天有人随机巡查，会导致商家被罚款

**洗澡热水供应时间**
8:00~22:00，澡堂有个小健身房和休息室

**一卡通充值情况**
学校发放一卡通（CPU 卡），生活热水 / 直饮水 / 门禁 / 图书馆 / 校车 / 食堂消费均使用此卡
- 充值方式：微信（东北师范大学企业公众号） / 支付宝（校园生活） / 学子卡转账（不常用）
- 圈存方式：目前食堂内大部分卡机可实现刷卡时自动圈存，也可以到自助终端手动圈存（数量少）
- 补办费用：20 元 / 张

**银行卡**
学校与交通银行合作，向学生发放学子卡（储蓄卡）

**现阶段门禁情况**
小程序自主申报，市内出行自动审核，跨市 / 跨省出行需向辅导员报备。审核通过后开通门禁 1 次出入权限，隔日作废

其他：

- 大一可以带电脑，可以直接考四六级
    - 若家庭条件或其他特殊原因不允许，图书馆有电子阅览室，每学期有一定免费额度
- 有免修 / 提前毕业政策，学霸请咨询学院
- 不强制收看青年大学习（可能看学院，反正我们学校常年在省榜里排不上号）
- **无**正版软件授权
- 净月校区水电不收费，本部电费按量计价，有免费额度且基本够用
- 宿舍每个位置限制功率 300 W
- 每人每天有 20 L 生活热水，约 2 L 直饮净化水（常温 / 热水），直饮水可充值，不过一般不需要。如果你丢过校园卡结果补办又给找回来了，那旧卡依然可以拿来刷水
- 挂科有一次补考机会，重修会计算学分，每个学生仅在课程计划外有 15 分免费学分，超出部分按 80 元 / 分计费，毕业时结算。
- 净月的娃们不需要扫雪，净月校区有两个托管物业，分别负责生活区和教学区，校园非常干净（本部不清楚，可能也不用吧）
- 宿舍阿姨们都很友善（至少我遇到的），所以也尽量对她们友好一些吧
- 每栋宿舍楼一楼有自贩机，可口可乐系（仅支持微信支付） + 农夫山泉系（微信 / 微信刷脸 / 支付宝 / 银联云闪付 / 农行掌银 / 京东支付）
- 生活设施分布
    - 二食堂一楼：超市、理发店、美甲店、水果店、书店、打印店、奶茶店、干果店、日杂店、文具店、糕点店、瑞幸咖啡、快递中心（疫情期间所有快递均在此集散）
    - 澡堂周边：小超市、理发店、水果店、打印店、洗衣店、修自行车 / 配钥匙
    - 一食堂：水吧、糕点店
- 绿化程度高，净月校区正门处有荷花池，本部有静湖
- 有虫，自行做好防虫措施
- 由于长春基建原因，停水是常事，且基本每年都会发生长时间停水，请做好储水措施
    - 2018 年夏曾连续停水 3 天
    - 2020 年 11 月的冻雨灾害直接导致长春几乎全市停水，净月停电，轻轨三号线停运
    - 2021 年 5 月曾连续停水超过 30 小时
- 一食堂旁有 ATM / 火车自助售票机

**Comment by Wonderdch** [comment]
本部以前大一是需要扫雪的，现在不知道怎么样了。
